ckanext/publicamundi/plugins.py

```diff
@@ -159,6 +159,10 @@ def before_map(self, mapper):
             m.connect(
                 '/api/publicamundi/dataset/export/{name_or_id}', 
                 action='dataset_export')
+
+            m.connect(
+                '/api/publicamundi/dataset/dcat/{name_or_id}',
```

Comment:
I suggest to name this as `dataset/export_dcat`, as this is also an export action.